Cargo.toml

```diff
@@ -49,7 +49,8 @@ env_logger = "0.8"
 mockito = "0.29"
 spectral = "0.6"
 test-case = "1.0.0"
-tokio = { version = "0.2", features = ["macros"] }
+tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
```

Comment:
The `rt-multi-thread` feature is required because `tokio` has been changed to always use the multi-threaded runtime, which is only available with the `rt-multi-thread` feature.
For more information, please see the `Runtime and Builder refactor` part of [tokio's official blog (Announcing Tokio 0.3 and the path to 1.0) ](https://tokio.rs/blog/2020-10-tokio-0-3).

Reply:
> tokio has changed to always use the multi-threaded runtime

As far as I understand, tokio still has current-thread runtime: https://docs.rs/tokio/1.2.0/tokio/runtime/struct.Builder.html#method.new_current_thread

Reply:
As you mentioned, we can use `tokio::runtime::Builder::new_multi_thread()` to use current-thread runtime, but `tokio::runtime::Runtime::new()` will always use multi-thread runtime.
https://docs.rs/tokio/1.2.0/tokio/runtime/struct.Runtime.html#method.new

Currently `dkregistry-rs` uses `Runtime` instead of `Builder`, so unless there is a strong reason to use a current-thread runtime, I think it's better to use the multi-thread runtime without changing the existing code.

Reply:
Ah, sorry. 

I haven't noticed that this is `dev-dependencies` change.